CONTRIBUTING.md

```diff
@@ -78,3 +78,33 @@ pnpm start
 That should start a webserver which will serve the docs on https://localhost:3000
 
 Any changes you make should be reflected in the browser. If you need to regenerate the API docs and find that your TSDoc isn't getting the updates, feel free to remove apps/docs/api. It will automatically regenerate itself when you run pnpm start again.
+
+## Publishing
+
+To publish a new version of the library, run
+
+```shell
+pnpm release # will run the build command, run the tests, and create a new version commit
```

Comment:
@himself65 actually, `pnpm release` also does call `pnpm publish`, so the flow is just:

```
pnpm new-llamaindex
pnpm release
git push
git push --tags 
```

Reply:
okay